### ISSUE TITLE
upload photo by data pointer

### ISSUE THREAD

**Issue by own2pwn** [dajobe/flickcurl#42]
is it possible to provide a pointer to a buffer rather than use filename to upload file?

**Comment by webbrain-one** [comment]
Yes — I opened [pull request #48](https://github.com/dajobe/flickcurl/pull/48), which adds support for uploading from a data pointer/buffer instead of requiring a filename.
